src/main/java/net/tropicraft/core/registry/BlockRegistry.java

```diff
@@ -193,6 +198,10 @@ public static void preInit() {
 		sifter = registerBlock(new BlockSifter(), Names.SIFTER);
 		flowerPot = registerBlockNoItem(new BlockTropicraftFlowerPot(), Names.FLOWER_POT, false);
 		bambooDoor = registerBlockNoItem(new BlockBambooDoor(), Names.BAMBOO_DOOR, false);
+
+		bongoSmall = registerBlock(new BlockBongoDrum(BongoSize.SMALL), Names.BONGO_SMALL);

```

Comment:
I feel like these should be variants and not separate blocks. Already have the enum, shouldn't be hard.